package.json

```diff
@@ -30,7 +30,8 @@
     "e2e-local": "CYPRESS_E2E_TEST_ENV=\"local\" cypress run",
     "e2e-local-open": "CYPRESS_E2E_TEST_ENV=\"local\" cypress open",
     "format": "prettier-eslint --write 'src/**/!(*.min).{js,jsx,ts,tsx,css,json}' 'e2e_tests/**/*.{js,jsx,ts,tsx,css,json}' 'scripts/**/*.{js,jsx,ts,tsx,css,json}' 'build_scripts/**/*.{js,jsx,ts,tsx,css,json}'",
-    "jest": "jest --coverage",
```

Comment:
This will affect CI, so we should either just add a new `jest-no-cov` script and leave the existing `jest` as it was, or else we should check what the change will mean/do on CI.